sdk/keyvault/azure-security-keyvault-certificates/CMakeLists.txt

```diff
@@ -52,6 +52,15 @@ target_include_directories(
       $<INSTALL_INTERFACE:include>
 )
 
+# Include shared source code
+# NOTE: Use shared-code only within .cpp files. DO NEVER consume the shared-code from header files.
+target_include_directories(
+  azure-security-keyvault-certificates
+    PRIVATE
+      $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/sdk/keyvault/azure-security-keyvault-shared/inc>
+      $<INSTALL_INTERFACE:include>
```

Comment:
Does `INSTALL_INTERFACE` make sense here? Why would we install the shared header(s) if they're only to be used from CPP files? It seems like we consume them only at build time.